docs/blog/_posts/2020-12-18-scala3-m3.md

```diff
@@ -0,0 +1,209 @@
+
+Note the difference from the Scala 2 scripting implementation. In Scala 2, we do not require the users to have a `main` method in their scripts due to it being too cumbersome to write. In Scala 3, thanks to the top-level definitions and the `@main` annotations, `main` methods are one-liners and hence are more suited for scripts.
+
+The documentation for this feature is available [here](https://dotty.epfl.ch/docs/usage/getting-started.html#scala-3-for-scripting).
+
+# Metaprogramming changes
+Metaprogramming-wise, the majority of work is focused on polishing the API and making it more uniform. The following notable metaprogramming API changes occurred between M2 and M3:
+
+- Add `Expr.asTerm` [#10694](https://github.com/lampepfl/dotty/pull/10694)
+- Add reflect `MatchCase` `TypeRepr` [#10735](https://github.com/lampepfl/dotty/pull/10735)
+- Rework reflect Symbol fields API [#10705](https://github.com/lampepfl/dotty/pull/10705)
+- Remove `Expr.StringContext.unapply` [#10675](https://github.com/lampepfl/dotty/pull/10675)
+- Rename `Liftable` to `ToExpr` and `Unliftable` to `FromExpr` [#10618](https://github.com/lampepfl/dotty/pull/10618)
+- Remove Unliftable[Unit]  [#10570](https://github.com/lampepfl/dotty/pull/10570)
+- Remove reflect.LambdaType [#10548](https://github.com/lampepfl/dotty/pull/10548)
+- Add `scala.quoted.Expr.unapply` as dual of `Expr.apply` [#10580](https://github.com/lampepfl/dotty/pull/10580)
+- Move `Quotes` as last parameter in `ExprMap.transform` [#10519](https://github.com/lampepfl/dotty/pull/10519)
+- Rework reflect Constant API [#10753](https://github.com/lampepfl/dotty/pull/10753)
+- Unify quoted.report and reflect.Reporting [#10474](https://github.com/lampepfl/dotty/pull/10474)
+- Fix #10359: Add GivenSelector to reflection API [#10469](https://github.com/lampepfl/dotty/pull/10469)
+- Rework reflect show API [#10661](https://github.com/lampepfl/dotty/pull/10661)
+- Fix #10709: Add missing level check before inlining [#10781](https://github.com/lampepfl/dotty/pull/10781)
+
+# Let us know what you think!
+If you have questions or any sort of feedback, feel free to send us a message on our
+[Gitter channel](https://gitter.im/lampepfl/dotty). If you encounter a bug, please
+[open an issue on GitHub](https://github.com/lampepfl/dotty/issues/new).
+
+
+## Contributors
+Thank you to all the contributors who made this release possible 🎉
+
+According to `git shortlog -sn --no-merges 3.0.0-M2..3.0.0-M3` these are:
+
+```
+TODO
+```
+
+If you want to get your hands dirty and contribute to Dotty, now is a good time to get involved!
+Head to our [Getting Started page for new contributors](https://dotty.epfl.ch/docs/contributing/getting-started.html),
+and have a look at some of the [good first issues](https://github.com/lampepfl/dotty/issues?q=is%3Aissue+is%3Aopen+label%3Aexp%3Anovice).
+They make perfect entry points into hacking on the compiler.
+
+We are looking forward to having you join the team of contributors.
```

Comment:
Is this paragraph still relevant at this point?

Reply:
It's always good to have new passionate people who are ready to contribute. I have reworded this paragraph to reflect the current situation.